### DISCUSSION TITLE
Cascade Delete?

### DISCUSSION THREAD

**Discussion by chantlong** [directus/directus#16939] [Archive]
Is that possible?

If Collection A has a Relationship with Collection B & C, I cannot delete Collection A.
How can I solve that issue?

**Comment by br41nslug** [comment]
You can set cascading deletes on the appropriate relational fields in your data-model 
![image](https://user-images.githubusercontent.com/9389634/209794281-b703c2ba-dc9b-4d84-a64c-11e52a59ba9b.png)

**Reply by chantlong** [reply]
Thank you!
